docs/examples/advanced_http.py

```diff
@@ -0,0 +1,12 @@
+from nameko.web import http
+
+
+class Service(object):
+
+    @http('GET', '/404')
+    def does_not_exist(self, value):
+        return 404, "Does Not Exist"
+
+    @http('GET', '/google')
+    def redirect(self,):
+        return 302, "", {'Location': 'https://www.google.com/'}
```

Comment:
Better example than 302 for a status that includes a header in the server response?